src/test/scala/nl.knaw.dans.easy.solr4files/ApplicationWiringSpec.scala

```diff
@@ -99,7 +99,8 @@ class ApplicationWiringSpec extends TestSupportFixture {
       // vaultStoreNames/stores can't be a file and directory so we need a stub, a failure demonstrates it's called
       override def initSingleStore(store: String) = Failure(new Exception("stubbed ApplicationWiring.initSingleStore"))
     }.initAllStores()
-    inside(result) { case Failure(Exception(msg, CompositeException(headCause :: _))) =>
+    inside(result) {
+      case Failure(WrappedCompositeException(msg, CompositeException(headCause :: _))) =>
```

Comment:
This test breaks because it checks the msg passed into the exception, I want to test against the created message.

Reply:
which message do you mean (don't know what the created message is...)? The one from `CompositeException`? Then you add `e @ ` before `CompositeException`, right?